Weatherman.py

```diff
@@ -0,0 +1,245 @@
+import os
+import sys
+
+
+def weatherMonth(year, month, res):
+    file_name = ''
```

Comment:
`weather_file_name`
why declare this variable with an empty value that makes no sense